frontend/demo/src/pages/Inbox/ConversationListItem/index.tsx

```diff
@@ -31,8 +31,8 @@ const mapStateToProps = (state: StateModel) => {
 const connector = connect(mapStateToProps, null);
 
 const FormattedMessage = ({message}: FormattedMessageProps) => {
-  if (message && message.content) {
-    return <>{message.content.text}</>;
+  if (message && message.content[0]) {
```

Comment:
This needs improvement but should be enough to unblock the ticket

Reply:
true! We should probably already add a follow up ticket (that discussion is relevant for the chat plugin as well)

Reply:
Since we will always have to deal with content lists, I think we need to reorganize the code so that the message renders as a wrapper that contains a mapped view of the content. Every single content block contains enough information to render it on its own.

Pseudo code:
```jsx
<Message>
{message.content.map((content) => renderMessageContent(content))}
</Message>
```

Reply:
The challenge for the data model is that as a default we have to accommodate sources that send multiple content pieces per message. This is why we had to make content a list, which is easy to handle with maps :)